mappings/net/minecraft/server/world/Players.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS uh net/minecraft/server/world/Players
```

Comment:
That name looks a bit too general for something used in just a few places

Reply:
Yup, this one is admittedly pretty bad.